### PR TITLE
Make LedgerNanoStatus locked an Option

### LINKED ISSUE #491: [Bug]: Ethereum app locked status returns wrong value when connected to the sdk.
### Issue description

Firefly tries to get the ledger status from the Ethereum Ledger app through `getLedgerNanoStatus` on the AccountManager object. The issue is that even if the app is locked, we get a `locked: false` property returned by the sdk. Could you guys look into this?

### Version

rc.35

### Expected behaviour

Whenever the Ethereum app is locked, we should get a `locked: true` response

### Actual behaviour

Locked Ethereum app returns a `locked: false` state
image

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

Steps to reproduce
1. Opt.: Set your ledger timeout to 1 min
2. Open the Ethereum app on Ledger
3. Wait until the ledger locks
4. The following status is returned from the sdk:

### Errors

None

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### PR DESCRIPTION
**Pull request by thibault-martinez**
Fixes https://github.com/iotaledger/iota-sdk/issues/491

### CODE REVIEW

File: sdk/src/client/secret/types.rs [hunk 1]
```diff
@@ -103,7 +103,7 @@ pub struct LedgerNanoStatus {
     /// Ledger is available and ready to be used.
     pub(crate) connected: bool,
     /// Ledger is connected and locked.
```
Comment (Thoralf-M) [inline sdk/src/client/secret/types.rs]: Probably good to update the comments that it's only Some() for Shimmer and IOTA app

Comment (thibault-martinez) [reply sdk/src/client/secret/types.rs]: Wanted to do this and forgot, thanks

Comment (thibault-martinez) [reply sdk/src/client/secret/types.rs]: Done

File: sdk/src/client/secret/types.rs [hunk 2]
```diff
@@ -120,7 +120,7 @@ impl LedgerNanoStatus {
         self.connected
     }
     /// Ledger is connected and locked.
```
Comment (Thoralf-M) [inline sdk/src/client/secret/types.rs]: Same here

Comment (thibault-martinez) [reply sdk/src/client/secret/types.rs]: Done

**Review by Thoralf-M — APPROVED** [review]

**Review by qrayven — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**